_data/navigation/doc_side_nav.yml

```diff
@@ -119,7 +113,27 @@ doc_side_nav:
             url: /docs/tutorials/basic/javascript.html
           - page: C++
             url: /docs/tutorials/basic/cpp.html
-  - page: Reference
-    url: /docs/reference/
+  - page: Client Libraries
+    url: '#client-libs'
+    sub_nav_id: client-libs
+    sub_nav:
+      - page: Java
+        url: '#lib-java'
+        sub_sub_nav_id: lib-java
+        sub_sub_nav:
+          - page: API Reference
+            url: https://spine.io/core-java/javadoc/client/index.html
+      - page: JavaScript
+        url: '#lib-js'
+        sub_sub_nav_id: lib-js
+        sub_sub_nav:
+          - page: API Reference
+            url: https://spine.io/docs/reference/javascript/index.html
```

Comment:
Now this link does not work. Are we planning to add it in the future?
The same for the `Dart API Reference`.

Reply:
They don't. There are going to be real things anytime soon.